src/Uno.UWP/ApplicationModel/PackageId.iOS.cs

```diff
@@ -0,0 +1,44 @@
+﻿#if __IOS__
+using Foundation;
+using SystemVersion = global::System.Version;
+
+namespace Windows.ApplicationModel
+{
+	public partial class PackageId
+	{
+		private const string BundleIdentifierKey = "CFBundleIdentifier";
+		private const string BundleShortVersionKey = "CFBundleShortVersionString";
+		private const string BundleVersionKey = "CFBundleVersion";
+
+		public string FamilyName => NSBundle.MainBundle.InfoDictionary[BundleIdentifierKey].ToString();
+
+		public string FullName => $"{NSBundle.MainBundle.InfoDictionary[BundleIdentifierKey].ToString()}_{NSBundle.MainBundle.InfoDictionary[BundleVersionKey]}";
+
+		public string Name => NSBundle.MainBundle.InfoDictionary[BundleIdentifierKey].ToString();
+
+		/// <summary>
+		/// Implementation based on <see cref="https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion"/>.
+		/// </summary>
+		public PackageVersion Version
+		{
+			get
+			{
+				var shortVersion = NSBundle.MainBundle.InfoDictionary[BundleShortVersionKey].ToString();
```

Comment:
```suggestion
				var shortVersion = NSBundle.MainBundle.InfoDictionary[BundleShortVersionKey].ToString() ?? "0.0";
```

Same for the one below. If for some reason the key does not exists.